xarray/tests/test_dataset.py

```diff
@@ -149,6 +149,21 @@ def test_repr_multiindex(self):
         print(actual)
         self.assertEqual(expected, actual)
 
+        # with scalar multiindex
+        data = data.sel(level_1='a')
+        expected = dedent("""\
+        <xarray.Dataset>
+        Dimensions:  (x: 2)
+        Coordinates:
+          * x        (x) MultiIndex
+          - level_1  <U1 'a'
```

Comment:
Now test fails here for Python2.7. Python2.7  seems to understand the dtype of `str` as `|S1` not `<U1`. This should be solved but is not related to scalar-level of MultiIndex.